global-styles.css

```
@@ -1,3 +1,12 @@
+:root {
+  --primary-purple: #846de9;
+  --turquoise: #24eaca;
+  --bg-purple-100: #fdfdff;
+  --bg-purple-200: #fbfaff;
+  --bg-purple-300: #eee8fe;
+  --text-purple-900: #160b46;
```

CSS 변수 이름에 색상 이름이 들어가는 것은 best practice에서 벗어날 것 같아요. 나중에 여러 색상 theme을 지원하게 될 때 문제가 될 소지가 있고요. CSS 변수 이름에는 용도만 표시해도 충분할 것 같습니다.

```suggestion
  --primary: #846de9;
  --secondary: #24eaca;
  --bg-100: #fdfdff;
  --bg-200: #fbfaff;
  --bg-300: #eee8fe;
  --text-900: #160b46;
```

---

반영했습니다!